File: src/main/java/no/nav/syfo/domain/dto/PDFTemplate.java
package no.nav.syfo.domain.dto;

public enum PDFTemplate {
    ARBEIDSTAKERE("arbeidstakere"),
    SELVSTENDIGNAERINGSDRIVENDE("selvstendignaeringsdrivende"),
    SYKEPENGERUTLAND("sykepengerutland");

    private String endpoint;

    PDFTemplate(String endpoint) {
        this.endpoint = endpoint;
    }

    @Override
    public String toString() {
        return this.endpoint;
    }
}

File: src/main/kotlin/no/nav/syfo/domain/dto/PDFTemplate.kt
package no.nav.syfo.domain.dto

enum class PDFTemplate(val endpoint: String) {
    ARBEIDSTAKERE("arbeidstakere"),
    SELVSTENDIGNAERINGSDRIVENDE("selvstendignaeringsdrivende"),
    SYKEPENGERUTLAND("sykepengerutland"),
    ARBEIDSLEDIG("arbeidsledig"),
    BEHANDLINGSDAGER("behandlingsdager"),
    ANNETARBEIDSFORHOLD("annetarbeidsforhold"),
    REISETILSKUDD("reisetilskudd"),
    GRADERT_REISETILSKUDD("gradertreisetilskudd");
}
